### PR TITLE
add set_alert_func

### PR DESCRIPTION
**Pull request by StefanSchoof**
Adds the set_alert_func.

I am unsure what do to with the second parameter of the callback. According to http://abyz.me.uk/rpi/pigpio/cif.html#gpioSetAlertFunc it is a Int with only 3 values. In Rest I think a enum is a good match, but I do not know what is the best way in this wrapper.

### CODE REVIEW

**Review by tadeegan — APPROVED** [review]
LGTM as long as you tested it.

**Comment by StefanSchoof** [comment]
Thanks, I tested this.

**Comment by tadeegan** [comment]
@martindeegan Can you give me merge permissions?

**Pull request merged**
